tests/test_models.py

```diff
@@ -0,0 +1,104 @@
+"""
+Models Test Case.
+
+Test the models of the application to be certain it's functioning well.
+"""
+
+import unittest
+
+from app import db, create_app
+from app.models import User, Base, BucketList, Items
+
+
+class UserModelTestCase(unittest.TestCase):
+    """
+    The class encompasses all test cases related to the Application's model.
+
+    I wrote simple tests to test the models.
+    """
+
+    def setUp(self):
+        """
+        Set up the application for testing.
+
+        The method 'setUp' simply starts the application in test mode.
+        """
+        self.app = create_app('testing')
+        self.app_context = self.app.app_context()
+        self.app_context.push()
+        db.create_all()
+
+    def tearDown(self):
+        """
+        Tear down method.
+
+        This method removes every information related to the test cases.
+        """
+        db.session.remove()
+        db.drop_all()
+        self.app_context.pop()
+
+    def test_user_base_inheritance(self):
+        """
+        Test user model.
+
+        Test if the User model inherits properties from the Base class.
+        """
+        user = User(username='test')
+        self.assertIsInstance(user, Base)
+
+    def test_bucketlist_base_inheritaance(self):
+        """
+        Test BucketList model.
+
+        Test if the BucketList model inherits properties from the Base class.
+        """
+        bucketlist = BucketList(
+            name="Bolaji's Personal"
+        )
+        self.assertIsInstance(bucketlist, Base)
+
+    def test_items_base_inheritance(self):
+        """
+        Test Items model.
+
+        Test if the Items model inherits properties from the Base class.
+        """
+        item = Items(
+            name="Jump off. a cliff"
+        )
+        self.assertIsInstance(item, Base)
+
```

Comment:
I think the tests  `test_user_base_inheritance` , `test_bucketlist_base_inheritaance` and `test_items_base_inheritance` are unnecessary. Checking if one class is a subclass of another is testing Python libraries, not the functionality you introduced.